doc/mako.5.scd

```diff
@@ -57,10 +57,23 @@ Supported options:
 	Default: dismiss
 
 *on-touch*=_action_
-	Performs the action when tapped via a touch device.
+	Performs the action when tapped via a touch device if the tap
+	duration is less than *long-press-duration*.
 
 	Default: dismiss
 
+*on-long-touch*=_action_
+	Performs the action when tapped via a touch device if the press
+	duration is greater or equal to *long-press-duration*.
+
+	Default: invoke-default-action
+
+*long-press-duration*=_time_
+	Specifies the cutoff time (in milliseconds) for a press to be
+	considered a long press.
+
+	Default: 500
```

Comment:
I'd prefer not to expose a config option for this for now. We can always add in the future if needed, but if there's no clear use-case for it, I prefer not to add something which might tie our hands later.